### PR TITLE
[MCC-170850] v1.5.0 - Expose ObjectExtensions to public

### PR DESCRIPTION
**Pull request by junshao**
- Made `object.ToJsonObject()` and `object.ToDynamic()` public, so that Rave code can utilize them.
- Version up to 1.5.0.

@art2003 @echen-mdsol Please review and merge if OK. Thanks

### CODE REVIEW

File: Mct.RaveCommon/ObjectExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        /// <returns>The dynamic object.</returns>
+        public static dynamic ToDynamic(this object target)
+        {
+            if (target == null) return null;
+            if (target is ExpandoObject) return target;
+
+            IDictionary<string, object> expando = new ExpandoObject();
+
+            foreach (PropertyDescriptor property in TypeDescriptor.GetProperties(target.GetType()))
+                expando.Add(property.Name, property.GetValue(target));
```
Comment (art2003) [inline Mct.RaveCommon/ObjectExtensions.cs]: Can we add local variable for  TypeDescriptor.GetProperties(target.GetType())

File: Mct.RaveCommon/ObjectExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        /// <returns></returns>
+        public static dynamic ToJsonObject(this object target)
+        {
+            if (target == null) return null;
+
+            var expandoObject = target as ExpandoObject;
+            if (expandoObject != null) return expandoObject;
+
+            var json = JsonConvert.SerializeObject(target, new StringEnumConverter());
+            var obj = JsonConvert.DeserializeObject<ExpandoObject>(json, new ExpandoObjectConverter());
```
Comment (art2003) [inline Mct.RaveCommon/ObjectExtensions.cs]: Should we handle null values, if any? (NullValueHandling.Ignore)

Comment (junshao) [inline Mct.RaveCommon/ObjectExtensions.cs]: Null properties will be sterilized to JSON `null`. So it should be fine.

File: Mct.RaveCommon/ObjectExtensions.cs (hunk 1 above)
Comment (junshao) [inline Mct.RaveCommon/ObjectExtensions.cs]: Thanks, changing

**Comment by junshao** [comment]
Comment addressed. Thanks. It's ready to be be merged now.

**Comment by art2003** [comment]
LGTM

**Pull request merged**